src/Page.js

```diff
@@ -516,9 +518,13 @@ Page.prototype.collectFrontMatter = function (includedPage) {
   } else {
     // Page is addressable but no front matter specified
     this.frontMatter = {
-      src: this.src,
-      title: this.title || '',
-      layout: LAYOUT_DEFAULT_NAME,
+      ...{
+        src: this.src,
+        title: this.title || '',
+        layout: LAYOUT_DEFAULT_NAME,
+      },
+      ...this.frontmatterOverride,
+      ...this.globalOverride,
```

Comment:
Here value specified by `site.json`, namely `this.title` would be overriden by `frontmatterOverride` and `globalOverride` if they specified a title. Would this be a concern?

Reply:
Probably not, it was discussed [here](https://github.com/MarkBind/markbind/issues/647#issuecomment-459240830) that it's easier to keep in consistent than specially handle it

Reply:
Ah i see, then line 515 and 517 should be done before 512? Otherwise it would override `frontmatterOverride` and `globalOverride` which is inconsistent when front matter is not specified.

Reply:
Sure, makes sense to move it down for consistency